week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,45 +1,47 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
+
 function mood() {
   let isHappy = true;
-
-  if (isHappy) {
+  if (!isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
+
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
-
-  if (isBigEnough) {
+  let isBigEnough = num;
+  if (isBigEnough >= num) {
```

Comment:
Again `semantics`.
- you assign `isBigEnough` to `num`
- then you compare them together, this comparison will always return true. ( think of the reason )
- I would rather solve it as follows: 
```
 let num = 10;
 let isBigEnough;

  if (isBigEnough >= 10 {
          ...
```